src/main/java/com/school/project/service/impl/UserServiceImpl.java

```diff
@@ -54,4 +61,11 @@ public User getUserById(Long id) {
         if (!byId.isPresent()) throw new UserNotFoundException("User with id :" + id + "is not found");
         return byId.get();
     }
+
+    @Override
+    public List<User> getAllUsers(){
+        return userRepository.findAll().stream()
+                .collect(Collectors.toList());
```

Comment:
I'm wondering why do we need streamApi here. Isn't it returned as List by default?